src/test/regress/expected/multi_citus_tools.out

```diff
@@ -629,6 +629,19 @@ SELECT citus_check_connection_to_node('localhost', :worker_2_port);
 (1 row)
 
 \c - - - :master_port
+SELECT * FROM citus_check_cluster_node_health() ORDER BY 1,2,3,4;
```

Comment:
seems like a good candidate to add to `arbitrary-configs` like `SELECT bool_and(success) citus_check_cluster_node_health()`

Reply:
This is a really good advice. I still have not figured out how to make my tests pass on all configs. Working on it

Reply:
Do you think that it is enough to see the result of the following?

```sql
SELECT bool_and(success) FROM citus_check_cluster_node_health();
```

I did not want to add complex checks as we have many configs now.

Reply:
Yes, that is what I had actually.